src/widgets/toggle-refinement/toggle-refinement.tsx

```diff
@@ -48,7 +49,9 @@ const renderer = ({
   isFirstRendering: boolean
 ) => {
   if (isFirstRendering) {
-    renderState.templateProps = prepareTemplateProps({
+    renderState.templateProps = prepareTemplateProps<
+      ToggleRefinementComponentTemplates
+    >({
       defaultTemplates,
```

Comment:
in some of the others you spread the defaultTemplates, but not here. Which pattern do you think is better? I'd like it to be the same in all widgets I think